srcpkgs/helden-software/patches/0001-remove-hsDebianMode-arg.patch

```diff
@@ -0,0 +1,34 @@
+From 0b5221d64f319df11a11b255a352105196a9e755 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Jan=20Christian=20Gr=C3=BCnhage?=
+ <jan.christian@gruenhage.xyz>
+Date: Sat, 25 Sep 2021 10:29:43 +0200
+Subject: [PATCH 1/3] remove '-hsDebianMode' arg
+
+---
+ etc/default/helden-software | 2 +-
+ usr/games/helden-software   | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/etc/default/helden-software b/etc/default/helden-software
+index b04bc7d..29fb662 100644
+--- a/etc/default/helden-software
++++ b/etc/default/helden-software
+@@ -1,2 +1,2 @@
+ JavaArgs="-Xms40m -Xmx256m"
+-HeldenArgs=-hsDebianMode
++HeldenArgs=""
+diff --git a/usr/games/helden-software b/usr/games/helden-software
+index 49d866b..9105ab5 100755
+--- a/usr/games/helden-software
++++ b/usr/games/helden-software
+@@ -1,6 +1,6 @@
+ #!/bin/sh
+ JavaArgs="-Xms40m -Xmx256m"
+-HeldenArgs=-hsDebianMode
```

Comment:
Is this actually what you want in a system-wide installation. It looks to me, that this has two modes, "Debian" for a system installation and "local" if it can write next to its jar. Since currently launching it will give this warning:
![image](https://user-images.githubusercontent.com/7373356/173227027-523cefcc-ad2b-414f-88f8-af57efb13c29.png)

Reply:
So that's what the Debian mode does, good to know. I couldn't find documentation for that parameter, so I turned it off. 

```text
void-packages on  helden-software-5.5.3_1 [$?⇕] took 13s 
❯ java -jar ~/Downloads/helden-deb/usr/lib/heldensoftware/helden5.jar --help
Der Parameter --help ist unbekannt!
Mit "/?" koennen alle moeglichen Parameter angezeigt werden.
Jun 13, 2022 2:13:05 PM helden.gui.ooOO.i:-1 o00000
WARNING: Ladezeit: 0.277081554

void-packages on  helden-software-5.5.3_1 [$?⇕] 
❯ java -jar ~/Downloads/helden-deb/usr/lib/heldensoftware/helden5.jar '/?'
---Parameterliste---
"-hsLocal"  setzt alle Pfade relative zum Jarfile, dieser Modus ist zB für USB-Sticks gedacht
"-hsLocalOff" schaltet den Localmodus wieder aus
"-ep" Durch diesen Parameter wird die Einstellungsdatei aus diesem Verzeichnis geladen.  z.B.  "-epd:\helden"
```

^ this doesn't list `-hsDebianMode` as a valid parameter anyway

In that case though, I'll remove that patch.